<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMX Panoramic Tomography User Interface</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/LNLS_logo_mini.jpg</normaloff>Images/LNLS_logo_mini.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(234, 234, 234)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="FilePathLabel">
        <property name="text">
         <string>Experiment Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ExperimentPathLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExperimentPathChooseButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/closed_folder.png</normaloff>Images/closed_folder.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="IMXGUINameLabel">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>IMX Panoramic Tomography User Interface</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LNLSLogoImageLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Images/LNLS_logo_mini.jpg</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="PanoramicFolderNameLabel">
        <property name="text">
         <string>Panoramic Folder Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PanoramicFolderNameLineEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Type panoramic folder name...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="NumberofScreensLabel">
        <property name="text">
         <string>Number of Screens (HxV)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NumberofScreensComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>2x1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2x2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3x1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3x2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3x3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="CloseGuiButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="SavePositionButton">
        <property name="text">
         <string>Save Position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="StartPanoramicTomographyButton">
        <property name="text">
         <string>Start Panoramic Tomography</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="SampleLengthLabel">
          <property name="text">
           <string>Sample Length (1 to 10 mm)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SampleLengthLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Type the bigger length your sample have...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QGridLayout" name="ImagesLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="HV1x1CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="HV3x3CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="HV3x3">
        <property name="text">
         <string>3x3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="HV1x3">
        <property name="text">
         <string>1x3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="HV2x1CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="HV1x1">
        <property name="text">
         <string>1x1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="HV3x2">
        <property name="text">
         <string>3x2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="HV2x1">
        <property name="text">
         <string>2x1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="HV3x1CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="HV1x2">
        <property name="text">
         <string>1x2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="HV2x2">
        <property name="text">
         <string>2x2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="HV2x3">
        <property name="text">
         <string>2x3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="HV1x3CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="HV1x2CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="HV3x1">
        <property name="text">
         <string>3x1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="HV3x2CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="HV2x2CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="HV2x3CheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionContact"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contacts</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>ExperimentPathChooseButton</tabstop>
  <tabstop>PanoramicFolderNameLineEdit</tabstop>
  <tabstop>NumberofScreensComboBox</tabstop>
  <tabstop>HV1x1CheckBox</tabstop>
  <tabstop>HV2x1CheckBox</tabstop>
  <tabstop>HV3x1CheckBox</tabstop>
  <tabstop>HV1x2CheckBox</tabstop>
  <tabstop>HV2x2CheckBox</tabstop>
  <tabstop>HV3x2CheckBox</tabstop>
  <tabstop>HV1x3CheckBox</tabstop>
  <tabstop>HV2x3CheckBox</tabstop>
  <tabstop>HV3x3CheckBox</tabstop>
  <tabstop>SavePositionButton</tabstop>
  <tabstop>CloseGuiButton</tabstop>
  <tabstop>ExperimentPathLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CloseGuiButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
